### DISCUSSION TITLE
Reference list of user Ids to each User in a Notification model

### DISCUSSION THREAD

**Discussion by YassinEldeeb** [prisma/prisma#8493] [Q&A] [answered]
That was a very silly question from me, I apologize...wish I can delete it but there's no way for doing that.

I can just make a different model like this
```prisma
model NotificationFromUser {
  id             String       @id @default(cuid())
  user           User         @relation(fields: [userId], references: [id], onDelete: Cascade)
  notification   Notification @relation(fields: [notificationId], references: [id])
  userId         String
  notificationId String
}
```
and then add it to `fromUsers` type to be `fromUsers      NotificationFromUser[]`

waiting for the production ready mongodb release cause I'm pretty new to sql databases (Postgres) and I'm making a lot of silly mistakes with it.

**Comment by ryands17** [comment, marked as answer]
@YassinEldeeb 👋 

So I assume this is solved?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
